File: Ana-Teresa-Schneider-Brandao/Semana_04/bilheteria.c
#include "bilheteria.h"

StatusIngresso calcular_ingresso(int idade, double *preco) {
    if (idade > 120) {
        return IDADE_INVALIDA;
    }
    if (idade <= 5) {
        *preco = 0.0;
    } else if (idade < 18) {
        *preco = 20.0;
    } else if (idade <= 60) {
        *preco = 40.0;
    } else {
        *preco = 20.0;
    }
    return OK;
}

File: julia_neiva_251023291/SEMANA_4/Bug_report/bilheteria.py
def calcular_ingresso(idade: int) -> float:
    if idade < 0 or idade > 120:
        raise ValueError("Idade inválida")
    if idade <= 5:
        return 0.0
    elif idade < 18:
        return 20.0
    elif idade < 60:
        return 40.0
    else:
        return 20.0